src/app/(params-search-params)/(client)/log/page.tsx

```
@@ -0,0 +1,19 @@
+import { Metadata } from "next";
+import Link from "next/link";
+
+export const metadata: Metadata = {
+  title: "Log",
+};
+
+type Props = {};
+export default function Log({}: Props) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)